portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACUtils.macro

```diff
@@ -453,6 +469,38 @@ definition {
 		Click(locator1 = "ACUtils#MODAL_CLOSE_BUTTON");
 	}
 
+	macro createABTest {
+		if (IsElementNotPresent(locator1 = "ACUtils#CREATE_TEST_BUTTON")) {
+			Click(locator1 = "ACUtils#AB_TEST_ICON");
+		}
+
+		Click(locator1 = "ACUtils#CREATE_TEST_BUTTON");
+
+		Click(locator1 = "ACUtils#AB_TEST_NAME_FIELD");
+
+		Type(
+			locator1 = "ACUtils#AB_TEST_NAME_FIELD",
+			value1 = "${titleAB}");
+
+		Select(
+			locator1 = "ACUtils#AB_GOAL_SELECTION",
+			value1 = "${goalAB}");
+
+		Click(locator1 = "ACUtils#SAVE_AB_TEST_BUTTON");
+	}
+
+	macro createABVariant {
+		Click(locator1 = "ACUtils#CREATE_VARIANT_BUTTON");
+
+		Click(locator1 = "ACUtils#AB_TEST_NAME_FIELD");
+
+		Type(
+			locator1 = "ACUtils#AB_TEST_NAME_FIELD",
+			value1 = "${variantAB}");
+
+		Click(locator1 = "ACUtils#SAVE_AB_TEST_BUTTON");
```

Comment:
Hey @joshitagaki, I think `Button.clickSave();` macro will work here. What do you think?